content/docs/buildpacks/language-family-buildpacks/nodejs.md

```diff
@@ -193,6 +193,36 @@ deprecated in Node Engine Buildpack v1.0.0. To migrate from using
 `buildpack.yml` please set the `BP_NODE_OPTIMIZE_MEMORY` environment variable
 mentioned above.
 
+## Using CA Certificates
```

Comment:
Much of this is already documented at a top level (https://paketo.io/docs/buildpacks/configuration/#ca-certificates). I think it would make sense to link to this and keep the Node specific behavior documented (NODE_OPTIONS).

Reply:
cool will do